src/pages/developers/evm/gateway.mdx

```diff
@@ -1,6 +1,14 @@
 Gateway is an interface that serves as a unified entry point for interactions
 between contracts on connected chains and universal apps on ZetaChain.
 
+<iframe
```

Comment:
The Figma diagram looks really cool, I'm wondering if they have any query params or options to remove the footer of the diagram that redirects to the actual Figma page, for one I don't think "Untitled" is a very fitting name.

<img width="1511" alt="image" src="https://github.com/user-attachments/assets/cecd3413-80cf-4fe0-a507-fa3affb7f5dc" />

But I also don't think it makes sense to display the names of the core contributors that are currently viewing the file.

<img width="1511" alt="image" src="https://github.com/user-attachments/assets/32daff3a-2dca-4bf9-9773-26b5a5c01be3" />

Reply:
Thanks for feedback! Removed footer and renamed the file [hide footer](https://github.com/zeta-chain/docs/pull/518/commits/0b8342d5e7505f3da4abe339ae3b40c357de4053). Removing viewers would be nice, but it's a Figma account config, so we can't really fix it in docs source. I'll ask the team if it's something we can change.